.github/workflows/build-release.yml

```diff
@@ -13,16 +13,19 @@ env:
 jobs:
   build-release:
     name: Build and push tag images
-    strategy: 
+    strategy:
       matrix:
         architecture: [amd64,ppc64le,arm64,s390x]
         platform: [linux]
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Set Env Tags
+    - name: Set Release Tag
       run: echo RELEASE_TAG=$(echo $GITHUB_REF | cut -d '/' -f 3) >> $GITHUB_ENV
 
+    - name: Set COMMIT_SHA/Version
+      run: echo COMMIT_SHA=$(git rev-parse HEAD) >> $GITHUB_ENV
```

Comment:
Any reason why we're running `git` instead of relying on the environment for the event that triggered this workflow (e.g. the same as we do in the previous step, which uses `$GITHUB_REF`, we could be using `$GITHUB_SHA`

Reply:
Further, I think we could probably skip this step entirely if `$GITHUB_SHA` is in the right format already. That is, we do the previous step for `$RELEASE_TAG` because the format of that value is not what we want, and we have to manipulate it to get the value we want. I'm guessing `$GITHUB_SHA` can probably go straight into your workflow_call, no?

Reply:
We are using `git` in other workflows so I thought we'd want to use it here as well, also since there are multiple SHA's in the GH ENV, it's confusing which one is actually `git rev-parse HEAD` value, and I didn't want there to be a mismatch. Are we sure we want `GITHUB_SHA` and not `COMMIT_SHA` or even `GITHUB_WORKFLOW_SHA`?

Reply:
COMMIT_SHA is what you've set, right? WORKFLOW_SHA is documented as the sha of the workflow file itself. GITHUB_SHA is what you want.

If you want to leave the `git` call, I'll leave it up to you. In practice, what you're doing is letting actions/checkout dictate what sha you have, and that defaults to GITHUB_SHA from what I can tell.

Should be fine. Just something to note.

Reply:
I'm fine for reducing code that we need to maintain, if we're sure `GITHUB_SHA` is what we want I'll change to that. I'll double check that this is the same as have we have in our make file that is executing `git re-parse HEAD` that's present on/in the binary.

Reply:
I verified that the sha's produced in our make file matches `GITHUB_SHA`

### Info In Binary
```
$ wget https://github.com/acornett21/openshift-preflight/releases/download/0.0.32/preflight-linux-amd64
$ ./preflight-linux-amd64 --version
preflight version 0.0.32 <commit: 124521d9840b619a404d16e2bf5eaa8d83335925>
```

### Info In `curl` in GHA
```
curl -s \
  --cert '/tmp/preflight-release.crt' \
  --key '/tmp/preflight-release.key' \
  --pass '***' \
  -H 'Content-Type: application/json' \
  -d '{"commit":"124521d9840b619a404d16e2bf5eaa8d8335925","enabled_for_testing":true,"name":"github.com/redhat-openshift-ecosystem/openshift-preflight","version":"0.0.32"}' \
  -X POST \
  'https://pyxis.preprod.api.redhat.com/v1/tools' | jq
```